### PR TITLE
Resolve ra/phi/phase discrepancy

### PR DESCRIPTION
**Pull request by EthanMarx**
Addresses #52, and another bug where testing set waveforms were being computed with `dec` for their `phi` and `psi` values.

Things to remember:

1. `phi` refers to the relative angle between the orientation of earth and the source. To back out a physical `ra` value, one only needs the additional information of the `gmst` (i.e. which direction the earth is facing at the time of the event). During training, we learn to predict `phi`. At inference time, we predict `phi`, and given the coalescence time, calculate the `gmst` and convert to physical `ra` by adding (or subtracting depending on how ml4gw has defined it)`phi`.   
2. `phase` refers to the phase of the waveform and has been added as an `inference_param`
3. `psi` refers to the polarization angle (i.e. relative h+ / hx content). Don't think this was an ambiguity but putting it here for clarity anyways.

### CODE REVIEW

**Comment by EthanMarx** [comment]
Also fixes a couple of config issues when running the sandbox pipeline via `pinto run`

File: projects/sandbox/pyproject.toml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [tool.pinto]
 steps = [
-    "data_generation:generate-background",
-    "data_generation:generate-waveforms",
-    "data_generation:generate-testing",
-    "train:train:coupling",
-    "sample_testing_set:sample_testing_set:coupling"
+    #"data_generation:generate-background",
```
Comment (deepchatterjeeligo) [inline projects/sandbox/pyproject.toml:3]: Undo these changes please

**Comment by deepchatterjeeligo** [comment]
This functionally resolves the issue we had. This is otherwise running end to end?

File: projects/sandbox/sampling/pyproject.toml [hunk 2]
```diff
@@ -11,6 +11,8 @@ torch = {version = "^1.12", source = "torch"}
 "mlpe.data" = {path = "../../../libs/data", develop  = true}
 "mlpe.trainer" = {path = "../../../libs/trainer", develop = true}
 "mlpe.logging" = {path = "../../../libs/logging", develop = true}
+"mlpe.injection" = {path = "../../../libs/injection", develop = true}
```
[the remark this answers is not in the document]
Comment (EthanMarx) [reply projects/sandbox/sampling/pyproject.toml:14]: It looks like we need it for importing the prior.
I'm seeing from 
`mlpe.injection.priors import sg_uniform`
in `samples.py`

[the remark this answers is not in the document]
Comment (deepchatterjeeligo) [reply projects/sandbox/sampling/pyproject.toml:14]: Right indeed. Sorry, I just looked at the diff.

**Comment by EthanMarx** [comment]
Yeah I was able to run end to end. Still having weird issues where the train log will not write.
